river/Root.zig

```diff
@@ -331,6 +333,7 @@ pub fn deactivateOutput(root: *Self, output: *Output) void {
         root.notifyLayoutDemandDone();
     }
     while (output.layouts.first) |node| node.data.destroy();
+    server.input_manager.reassignDevices();
```

Comment:
wlroots will automatically detach cursor from output if it's gone from the layout, so this is probably not necessary?

Reply:
I don't think wlroots does actually, at least not wlroots 0.17.

Reply:
Rather, yes wlroots has a listener for handling an output being removed from the layout but this handler does not do anything with output mappings.

Reply:
Correct, river will segfault after an output with a device  mapped to it is removed without this.